pkg/utils/hwaddr/hwaddr.go

```diff
@@ -15,49 +15,27 @@
 package hwaddr
 
 import (
-	"fmt"
+	"math/rand"
```

Comment:
do you foresee any problem with the deterministic behavior of "math/rand/ vs "crypt/rand"?
https://flaviocopes.com/go-random/

Reply:
Thanks! `crypto/rand` seems better here.

Reply:
done!